external/kafka/src/main/scala/org/apache/spark/streaming/kafka/KafkaRDD.scala

```diff
@@ -219,6 +219,7 @@ class KafkaRDD[
           null.asInstanceOf[R]
         } else {
           requestOffset = item.nextOffset
+          log.info("fetched message")
```

Comment:
Simply additional message... to aid in debugging when streaming app was not working with sparrow